server/models/account/user.ts

```diff
@@ -431,7 +486,10 @@ export class UserModel extends Model<UserModel> {
   }
 
   static loadById (id: number): Bluebird<MUserDefault> {
-    return UserModel.findByPk(id)
+    return UserModel.scope([
+      ScopeNames.WITH_VIDEOCHANNELS,
+      ScopeNames.WITH_STATS
```

Comment:
I think it would be better to include stats on demand (for example with a query parameter `?includeStats=true/false`

Reply:
`withStats`? That way, it matches the scope name.

Reply:
:+1: